tests/bluetooth/host/id/bt_id_create/src/main.c

```diff
@@ -215,3 +215,36 @@ ZTEST(bt_id_create, test_create_id_valid_input_address)
 	zassert_mem_equal(&bt_dev.id_addr[new_id], BT_STATIC_RANDOM_LE_ADDR_1, sizeof(bt_addr_le_t),
 			  "Incorrect address was set");
 }
+
+/*
+ *  Test creating a new public identity.
+ *
+ *  Constraints:
+ *   - A valid address of type public is used
+ *   - Input IRK is NULL
+ *   - 'BT_DEV_ENABLE' flag is set in bt_dev.flags
+ *
+ *  Expected behaviour:
+ *   - The public address is loaded to bt_dev.id_addr[BT_ID_DEFAULT]
+ *   - bt_dev.id_count is incremented
+ */
+ZTEST(bt_id_create, test_public_address)
```

Comment:
oh no
![image](https://github.com/zephyrproject-rtos/zephyr/assets/68428270/da6785fc-8a81-44db-878e-2600e89d5a9b)

See this branch for how to fix the test.
https://github.com/jori-nordic/zephyr/tree/49f1414d9dc74eb2fc3e23fc6ec4d23efffb4108/./

Reply:
edit: commit in question https://github.com/jori-nordic/zephyr/commit/cc2c1611654bcf77a7490418b4dcfdc4ae4a3341

Reply:
Thanks for taking care of that! I was wondering how to fix those tests.